bin/npmbuild-script.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+const fs = require("fs");
+const DIR = "./build";
+
+fs.readdir(DIR, (err, files) => {
```

Comment:
For most of these `fs` routines there's a `*Sync` version. So `fs.readdirSync`. These don't require callbacks. It's not something you should use in a server (it will block the process), but for scripts like this it's perfect.

Reply:
Hi, @ianb. Thank you for this. Woking on this issue really taught me a lot. I know it is supposed to be easy, but for me, it took me 3 weeks.